azure-pipelines.sds.pr-release.yml

```diff
@@ -36,10 +45,43 @@ stages:
               chartPath: ${{ variables.chartPath }}
               chartValuesPath: ${{ variables.chartPath }}
 
+      - job: UnitTests
+        displayName: "Unit Tests"
+        steps:
+          - checkout: self
+          
+          - template: stages\dotnet-tests\unit-tests.yml@azTemplates
+            parameters:
+              nugetConfigPath: ${{ variables.app_path }}
+              unitTestProjectPath: "${{ variables.app_path }}/${{ variables.app_path }}.UnitTests"
+              coverletCoverageExclusions:
+                - "[${{ variables.app_path }}.*Tests?]*"
+                - "[${{ variables.app_path }}]Startup"
+                - "[*]${{ variables.app_path }}.Common.*"
+                - "[Testing.Common]*"
+
+      - job: SonarCloud
+        displayName: SonarCloud Test Analysis
+        steps:
+          - template: templates/sonarCloud/prepare.yml@azTemplates
+            parameters:
+              sonarCloud_ExtraProperties: |
+                sonar.cs.opencover.reportsPaths=$(Common.TestResultsDirectory)\Coverage\coverage.opencover.xml
+                sonar.coverage.exclusions=${{ variables.sonar_exclusion }}
+
+          - bash: |
+              cd BookingsApi/BookingsApi.UnitTests
+              dotnet build BookingsApi.UnitTests.csproj
+            displayName: Build .NET Project(s)
```

Comment:
should this be a dotnet task?
Does this not already happen in the unit test job above? could merge?